test/config/examples/cifar10-pytorch.yml

```diff
@@ -14,7 +14,6 @@ assessor:
 trial:
   codeDir: ../../../examples/trials/cifar10_pytorch
   command: python3 main.py --epochs 1 --batches 1
-  gpuNum: 1
```

Comment:
Why remove gpuNum?

Reply:
It looks the test environment is used by two test suites sometime. So if set it to 1, NNI will wait another finished, but it causes timeout error. Remove this setting, it lets test cases to use default GPU, not wait each other, and reduced failure chance.

Reply:
We need to add examples to test gpuScheduler in NNI, if remove this configuration, we could not test gpu related functions in pipeline.

Reply:
ok, I will add it back. But it's test is very limited.